php-frontend/src/main/java/org/sonar/php/tree/symbols/SymbolVisitor.java

```diff
@@ -545,6 +551,14 @@ private void usageForNamespaceName(NamespaceNameTree namespaceName, Symbol.Kind
 
   @Override
   public void visitMemberAccess(MemberAccessTree tree) {
+    memberStatementDepth++;
+    if (memberStatementDepth >= MEMBER_ACCESS_MAXIMUM_DEPTH) {
+      if (LOG.isDebugEnabled()) {
+        LOG.debug("visitMemberAccess at \"{}...\" has reached the maximum depth of {}, switching to the next statement",
```

Comment:
We unwrap the chain from the end, so the current text pointer will be the 100th from the end and not from the start, and using it as a marker doesn't seem useful to me